Code/GraphMol/Chirality.cpp

```diff
@@ -118,16 +118,13 @@ namespace RDKit{
       }
     }
 
-    void iterateCIPRanks(const ROMol &mol, DOUBLE_VECT &invars, UINT_VECT &ranks,bool seedWithInvars){
+    void iterateCIPRanks(const ROMol &mol, DOUBLE_VECT &invars,
```

Comment:
Though the changes in this function seem to make sense after a brief review, I don't think they are connected to the const std::string fix that's the actual purpose of the PR.